docs/docs/guide/10-debug.md

```diff
@@ -171,13 +176,13 @@ Within the debugger shell create a breakpoint that will be triggered when the
 
 From a different terminal use the `hellod` binary to call the query:
 
-```bash
+```
 hellod query hello say-hello bob
 ```
 
 A debugger shell will be launched when the breakpoint is triggered:
 
-```
+```go
```

Comment:
The `go` syntax was removed to avoid issues when `mdgofmt` runs

Reply:
Thanks for letting me know. Removed.